command.go

```diff
@@ -103,11 +103,14 @@ func Auth(secret string) (*Command, error) {
 	return &Command{[]byte("AUTH"), nil, []byte(secret)}, nil
 }
 
-// Register creates a new Command to add a topic/channel for the connected nsqd
-func Register(topic string, channel string) *Command {
-	params := [][]byte{[]byte(topic)}
-	if len(channel) > 0 {
-		params = append(params, []byte(channel))
+// Register creates a new Command to add a topic/channel with state for the connected nsqd
+func Register(topic string, channel string, paused ...int) *Command {
```

Comment:
Should probably do the same here? (send a JSON-serialized dictionary of key/values as the third argument)